californium-core/src/main/java/org/eclipse/californium/core/network/UdpMatcher.java

```diff
@@ -74,7 +74,7 @@
 /**
  * A Matcher for CoAP messages transmitted over UDP.
  */
-public final class UdpMatcher extends BaseMatcher {
+public class UdpMatcher extends BaseMatcher {
```

Comment:
I can't see, that OSCORE is "extending" the `UdpMatcher`. Can you point me to the extension.

Reply:
You are correct. It is not used anymore.